added FAQs/Help page, added routing and made it look semi-presentable…

taranchana opened the pull request:
… (more work to be done, but good enough for initial commit)

EsamKhalid commented:
The code looks great Taran. The only thing that I would suggest adding would be to add a link to the FAQ page on the navbar in order to make it easier for the user to find which makes the website easier to navigate. Once that has been added I will be more than happy to merge as the rest of the page looks fine.  Good work.

taranchana commented:
Hi so I've updated the t&cs page, updated the faq page to be in line with everything else and the Faq and footer have been modified to fit a simplified approach

jiajoestar commented:
good work taran! i'm happy with the T&Cs and FAQs pages and changes made to the footer, contact and about page. nice!

jiajoestar reviewed (commented):
good work taran! i'm happy with the T&Cs and FAQs pages and changes made to the footer, contact and about page. nice!
